app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CategoriesController < ApplicationController
+  DEFAULT_LIMIT = 16
+  def show
+    return category_not_present unless category
+    category_products = category.products
+    @products = category_products.paginate(
+      page: params[:page], per_page: DEFAULT_LIMIT)
```

Comment:
**Nitpick - Code Quality**
Remember that when breaking an argument list, each argument should be placed on its own line.

For example
```
some_method(
      i_am_a_long_variable_name_that_will_never_fit_on_one_line_with_others,
      two,
      three
    )
```